pyproject.toml

```diff
@@ -3,15 +3,16 @@ name = "sktime"
 version = "0.14.0"
 description = "A unified framework for machine learning with time series"
 authors = [
-    {name = "Franz Király"},
-    {name = "Tony Bagnall"},
+    {name = "Franz Király", email = "f.kiraly@ucl.ac.uk"},
+    {name = "Tony Bagnall", email = "ajb@uea.ac.uk"},
     {name = "Markus Löning"},
-    {name = "Martin Walter"},
+    {name = "Martin Walter", email = "mwalter@sktime.org"},
 ]
 maintainers = [
-    {name = "Franz Király"},
+    {name = "sktime developers", email = "info@sktime.org"},
+    {name = "Franz Király", email = "f.kiraly@ucl.ac.uk"},
```

Comment:
I think we should be using the `sktime.org` emails. This is a bit of a relic.

The pattern is `fkiraly@sktime.org` etc, I think.
(@aiwalter is maintaining the email acct)